bakerydemo/templates/blocks/image_block.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailimages_tags %}
 
 <figure>
-    {% image self.image fill-600x338 loading="lazy" %}
+    {% picture self.image format-{avif,webp,jpeg} fill-{400x220,600x338} sizes="(max-width: 768px) 200px, 900px" loading="lazy" %}
```

Comment:
I chose 200px here for the mobile 'size' because of pixel density. When testing, if I made this number larger, the browser would choose the larger (600x338) image instead when emulating a mobile.